src/i18n/de/common.ts

```diff
@@ -12,7 +12,8 @@ const common: CommonMessages = {
   'common.amount': 'Amount - DE',
   'common.coin': 'Coin - DE',
   'common.password': 'Passwort',
-  'common.memo': 'Memo - DE'
+  'common.memo': 'Memo - DE',
+  'common.refresh': 'AKTUALISIERUNG'
```

Comment:
Capitalize texts w/ CSS (via `text-transform`) in a component, but not in `i18n` sources.